Dockerfile

```diff
@@ -2,17 +2,17 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS
 WORKDIR /go/src/github.com/openshift/cloud-credential-operator
 COPY . .
 ENV GO_PACKAGE github.com/openshift/cloud-credential-operator
+ARG TARGETPLATFORM
```

Comment:
Who sets this value?

Reply:
The docs say this is automatically set https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

And from my testing, buildah was setting it properly too https://github.com/containers/buildah/issues/1368

Reply:
I don't think this is supported in OSBS though.  But there is also no reason to limit the Mac and Windows binaries to just the amd64 payload though.